docs/source/how-to/refresh.md

```diff
@@ -0,0 +1,124 @@
+# Refreshing user authentication
+
+JupyterHub has a mechanism called [`refresh_user`](inv:jupyterhub:py:method#jupyterhub.auth.Authenticator.refresh_user) that is meant to 'refresh' information from the Authentication provider periodically.
```

Comment:
```suggestion
JupyterHub has a mechanism called [`refresh_user`](inv:jupyterhub:py:method#jupyterhub.auth.Authenticator.refresh_user) that is meant to _refresh_ information from the Authentication provider periodically.
```
I think italics should look better when rendered:
![image](https://github.com/user-attachments/assets/0e8420cf-51ac-4657-a65c-fcb8ef4fc1a5)